src/views/ResetPassword/ResetPassword.scss

```diff
@@ -0,0 +1,30 @@
+.passwordReset{
+    display: flex;
+    height: 100vh;
+    width: 100vw;
+    justify-content: center;
+    align-items: center;
+
+    h3{
+        position: absolute;
+        margin-top: 40px;
+    }
+
+.form{
+        border: .5px solid #88888810;
+        min-width: 200px;
+        width: 500px;
+        height: 500px;
+        border-radius: 10px;
+        background-color: white;
+        box-shadow: 1px 2px 8px -1px rgba(0,0,0,0.56);
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        form {
+            margin-top: 50%;
+            transform: translateY(-50%);
+        }
+    }
+}
```

Comment:
Great job @Hadeneekeh. I love your implementation and it works really well as I have been able to change my password and still log in successfully. Please add a new line to the end of this file and you're good to go.